Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -671,15 +673,20 @@ def get_time_range(last_fetch: Union[str, None] = None, time_range_start=FETCH_T
     return {'range': {time_field: range_dict}}
 
 
-def execute_raw_query(es, raw_query, index=None):
+def execute_raw_query(es, raw_query, index=None, size=None, page=None):
     try:
         raw_query = json.loads(raw_query)
     except Exception as e:
         demisto.info(f"unable to convert raw query to dictionary, use it as a string\n{e}")
 
     body = {"query": raw_query}
-    response = es.search(index=index or FETCH_INDEX, body=body)
-    return response
+    requested_index = index or FETCH_INDEX
+
+    try:
+        return es.search(index=requested_index, body=body, size=size, from_=page)
+    except Exception as e:
+        demisto.debug(f'got an error when searching with {body}, error:\n{e}')
+        return es.search(index=requested_index, body=raw_query, size=size, from_=page)
```

Comment:
looks like in the case that raw_json is provided and not meant to be wrapped we're relying on an exception here. 
Lets do the following instead

```
def execute_raw_query(es, raw_query, index=None, size=None, page=None):
    try:
        raw_query = json.loads(raw_query)
        if raw_query.get('query'):
            demisto.debug('query provided already has a query field. Sending as is')
            return es.search(index=requested_index, body=raw_query, size=size, from_=page)
    except TypeError as e:
        demisto.info(f"unable to convert raw query to dictionary, use it as a string\n{e}")

    body = {"query": raw_query}
    requested_index = index or FETCH_INDEX

    return es.search(index=requested_index, body=body, size=size, from_=page)

```

Reply:
looks good, but the catch of the exception should be i think `ValueError` and not `TypeError`, I will catch them both